### ISSUE TITLE
Change headerTitle dynamically

### ISSUE THREAD

**Issue by johnmikel** [Flyer53/jsPanel4#104]
Hi, I'm using jsPanel4 with VueJS and I was wondering if it is possible to change the headerTitle dynamically? I'm sort of mimicking the email functionality of gmail and this seems like the best library to use at the moment (great work, by the way).

What I want to do is this: I have a dropdown inside the content of jsPanel and after selecting the dropdown, the headerTitle should change accordingly. I tried doing it but it doesn't work.

I also checked if you have a refresh (like nextTick()) implementation for it as well but I can't find it. Is this possible?

Thanks for your reply!

**Comment by Flyer53** [comment]
Hi there,
and thanks for the praise 😄 

Take a look at the method [setHeaderTitle()](https://jspanel.de/api.html#methods/setHeaderTitle)

Should do the trick ...

**Comment by johnmikel** [comment]
Thanks, I forgot to add that I found that function 5 minutes after asking my question :)) but now, I am trying to implement it - it is just so hard! (I spent like an hour doing it and I still can't solve it).

So basically, I'm using this repo https://github.com/houzw/simple-vue-components/tree/master/jsPanel and the thing is, he is using a vue-directive. How do I call setHeaderTitle from there on?

You're welcome. Thanks for the library as well!
